spec/built-in-tiles-spec.coffee

```diff
@@ -48,6 +48,65 @@ describe "Built-in Status Bar Tiles", ->
         runs ->
           expect(fileInfo.currentPath.textContent).toBe 'sample.txt'
 
+    describe "when buffer's path is clicked", ->
+      it "copies the absolute path into the clipboard if available", ->
+        waitsForPromise ->
+          atom.workspace.open('sample.txt')
+
+        runs ->
+          fileInfo.currentPath.click()
+          expect(atom.clipboard.read()).toBe fileInfo.getActiveItem().getPath()
+
+    describe "when path of an unsaved buffer is clicked", ->
+      it "copies the 'untitled' into clipboard", ->
+        waitsForPromise ->
+          atom.workspace.open()
+
+        runs ->
+          fileInfo.currentPath.click()
+          expect(atom.clipboard.read()).toBe 'untitled'
+
+    describe "when buffer's path is not clicked", ->
+      it "doesn't display a path tooltip", ->
+        jasmine.attachToDOM(workspaceElement)
```

Comment:
Because this test needs to access DOM element and check if it doesn't exist while the test above is just checking clipboard and doesn't have to work with DOM at all.